testsuite/src/it/java/io/quarkus/ts/startstop/CodeQuarkusTest.java

```diff
@@ -200,7 +202,9 @@ public void supportedExtensionsSubsetB(TestInfo testInfo) throws Exception {
 
     @Test
     public void supportedExtensionsSubsetC(TestInfo testInfo) throws Exception {
-        testRuntime(testInfo, supportedEx.get(2), MvnCmds.MVNW_DEV);
+        List<CodeQuarkusExtensions> supportedExtensionsSubsetC = supportedEx.get(2);
+        supportedExtensionsSubsetC.add(CodeQuarkusExtensions.QUARKUS_RESTEASY);
+        testRuntime(testInfo, supportedExtensionsSubsetC, MvnCmds.MVNW_DEV);
```

Comment:
I miss the point here. I guess what we want is to verify the third group of supported extensions, but ensuring we have the resteasy extension which contains the code starter, but I missed why we want to verify this use case as the behaviour of the old test case has changed.
If this use case is relevant, I would add a new test case and keep the `supportedExtensionsSubsetC` as it was, but maybe a comment to clarify this would help too.

Reply:
That's coming from Kyrylo's PR, I will add comment to explain the need for this.
tl;dr: resteasy or spring-web extension is needed to provide index.html file we are checking in the tests.

Reply:
Then, all the combinations must contain either the resteasy or spring-web? Otherwise, I will add a comment to clarify this for others in the future too.

Reply:
subsetA and subsetB contain the needed extensions, so just subsetC was needed to be enhanced "manually"